### PR TITLE
Create initial trajectory package

### PR DESCRIPTION
**Pull request by ferdianjovan**
Trajectory package will be an online stitching human poses that will be used for initial for in WP5.
This is the initial package.

### CODE REVIEW

**Comment by lucasb-eyer** [comment]
So this should become the common output of the trackers?

File: human_trajectory/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+## Specify libraries to link a library or executable target against
+# target_link_libraries(human_trajectory_node
+#   ${catkin_LIBRARIES}
+# )
+
+#############
+## Install ##
+#############
+
```
Comment (cdondrup) [inline human_trajectory/CMakeLists.txt:113]: Please define install targets for release.

**Comment by cdondrup** [comment]
Some of the files make use of `cmake_modules` which is new in indigo. Sorry to be a pain, but can you open the PR again against the indigo-devel branch? I will make indigo-devel the main branch as well.

**Comment by kunzel** [comment]
@lucasb-eyer, Not exactly. We will leave the output of the trackers as it is. The idea is rather to postprocess the the individual poses reported by the tracker and generate a single trajectory message. Thereby a trajectory becomes a 'first-class citizen' which make things easier at a higher level (WP4/WP5).

**Comment by lucasb-eyer** [comment]
I see. In the mid-term future, we'll augment the tracker outputs with pointers back to the detections, so that it's possible to get to the (say upper-body) detections which belong to a track. This is essential for me to do anything on top of a track, and #116 asks for something similar. Please have this in mind when designing the code, at least so that it could be added later without too much fuss.

ping @vonovak

**Comment by hawesie** [comment]
+1 for linking in detections to tracks and allowing this to be found from the trajectories too

**Comment by lucasb-eyer** [comment]
Even a pointer from the trajectories to tracks would be enough so one could go trajectories->tracks->detections.

**Comment by kunzel** [comment]
@lucasb-eyer When you say 'pointer' do you mean unique IDs that you will introduce? We can include such a field/list in the trajectory message, so that we can refer back to the detections. Please let us know when you start working on this, so we can include this in the message definition.

**Comment by lucasb-eyer** [comment]
Yeah, something like that. And I'll make sure to let you know.

**Pull request closed without merging**